config.json

```diff
@@ -20,6 +20,17 @@
       ]
     },
     {
+      "uuid": "dc4ad67d-0fe4-0880-5a13-3aebb8759bcc3095de1",
+      "slug": "two-fer",
+      "core": false,
```

Comment:
> My only concern with having this as a bonus rather than a core exercise, is it contains info introducing a couple of things, stubs and example tests, and that these may get bypassed if the exercise doesn't need completing

Oh! Yeah, totally. That is an excellent point. You just changed my mind.